"moonwave dev" cannot run due to npm security patch

Due to some security patches npm cannot run `moonwave dev`, throwing an error reading `Error: spawn EINVAL`

![Code_roKtRR1N6X](https://github.com/evaera/moonwave/assets/96698128/dc563ff9-ebaf-4be2-8ba2-c1ed0e6825f3)

A workaround for this is to revert to 16.20.2. However, after running `moonwave dev` in that version, I get this error:

![Code_lQZBWorKXY](https://github.com/evaera/moonwave/assets/96698128/48414cdf-9caf-4a8f-bbc6-18eaf45a547b)

```
[ERROR] Error: Command failed: "C:\Users\c5599\AppData\Roaming\nvm\v16.20.2\node_modules\moonwave\dist\bin\moonwave-extractor-1.1.2.exe" extract "C:/Users/c5599/Documents/GitHub/Slap-Battles-Rojo-Transfer/src" --base "C:\Users\c5599\Documents\GitHub\Slap-Battles-Rojo-Transfer"

thread 'main' has overflowed its stack

    at ChildProcess.exithandler (node:child_process:402:12)
    at ChildProcess.emit (node:events:513:28)
    at maybeClose (node:internal/child_process:1100:16)
    at Socket.<anonymous> (node:internal/child_process:458:11)
    at Socket.emit (node:events:513:28)
    at Pipe.<anonymous> (node:net:301:12)
```

Is there a workaround for this? If not, hope this gets patched soon 🙏🏻

--- a/cli/src/commands/build.ts
+++ b/cli/src/commands/build.ts
@@ -29,6 +29,7 @@ export default async function buildCommand(args: Args) {
     const command = "npm" + (process.platform === "win32" ? ".cmd" : "")
     const spawnOptions: SpawnOptions = {
       cwd: tempDir,
+      shell: true,
       stdio: "inherit",
     }
 

--- a/cli/src/commands/dev.ts
+++ b/cli/src/commands/dev.ts
@@ -62,6 +62,7 @@ export default async function devCommand(args: Args) {
         ["run", "start"],
         {
           cwd: tempDir,
+          shell: true,
           stdio: "inherit",
         }
       )

